src/lib/containers/home_page/featured-data/FeaturedDataTabs.tsx

```diff
@@ -7,6 +7,8 @@ import getColorPallette from '../../../containers/ColorGradient'
 export type FeatureDataTabProps = {
   title: string, // type of data being shown, used for the tab title and explore all button
   icon?: string,
+  exploreObjectType?: string,
+  explorePagePath?: string,
```

Comment:
Now you can define the explore object type and page path per tab (it's not the same in dHealth, but will remain the same in AMP-AD)